lms/djangoapps/mobile_api/models.py

```diff
@@ -1,3 +1,28 @@
 """
-A models.py is required to make this an app (until we move to Django 1.7)
+ConfigurationModel for the mobile_api djangoapp.
 """
+# pylint: disable=no-member
```

Comment:
It is not. I originally copied the from another ConfigurationModel. I moved the pylint to where it should be.